### ISSUE TITLE
It seems an error occurred while parsing JSON 

### ISSUE THREAD

**Issue by bluesatan0-0** [leoding86/pixiv-omina#38]
![image](https://user-images.githubusercontent.com/29154434/96801963-6bd9e480-143b-11eb-83d3-5c71514531b5.png)

**Comment by leoding86** [comment]
Could you provide me the work you had tried to download?

**Comment by bluesatan0-0** [comment]
![image](https://user-images.githubusercontent.com/29154434/97095610-3845c700-1694-11eb-800f-537229b330de.png)
https://www.pixiv.net/users/28061721

**Comment by leoding86** [comment]
I can't reproduce the issue. I think it is maybe your network issue.
Can you open the url 'https://www.pixiv.net/ajax/user/28061721/profile/all' is your browser? The browser should output content like below.
```json
{
   "error":false,
   "message":"",
   "body":{
      "illusts":{
         "84776189":null,
         "83758121":null,
         "83125332":null,
         "82452743":null,
         "82366612":null,
         "77715139":null,
         "75566834":null,
         "74138731":null,
         "73162709":null,
         "73147024":null,
         "72527093":null,
         "71980185":null,
         "67340508":null,
         "66505842":null,
         "66220039":null,
         "66102900":null,
         "65449851":null,
         "65374045":null
      },
      "manga":[
         
      ],
      "novels":[
         
      ],
      "mangaSeries":[
         
      ],
      "novelSeries":[
         
      ],
      "pickup":[
         {
            "type":"fanbox",
            "deletable":false,
            "draggable":true,
            "userName":"Marugaogaski",
            "userImageUrl":"https:\/\/i.pximg.net\/user-profile\/img\/2019\/12\/06\/22\/51\/27\/16648694_754b62da960761db9c01d855e4e56768_170.png",
            "contentUrl":"https:\/\/www.pixiv.net\/fanbox\/creator\/28061721?utm_campaign=www_profile\u0026utm_medium=site_flow\u0026utm_source=pixiv",
            "description":"#\u7121\u4fee\u6b63\u7248Uncencored#\u9ad8\u89e3\u50cf\u5ea6\u7248HighRes#\u304a\u8535\u5165\u308a\u753b\u50cfLimited\r\n\r\n\u307b\u305e\u307c\u305d\u30683DCG\u753b\u50cf\u3092\u51fa\u529b\u3057\u3066\u3044\u308bmarugaogaski\u306b\u304a\u5e03\u65bd\u3001\u652f\u63f4\u3092\u3057\u305f\u3044\u3068\u3044\u3046\u4eba\u5411\u3051\u306e\u30da\u30fc\u30b8\u3067\u3059\u3002\r\nThis page is for people who want to support marugaogaski.\r\n\r\n\u30d7\u30e9\u30f3\u5185CG\u306fpixiv\u3067\u516c\u958b\u3057\u3066\u3044\u308b\u753b\u50cf\u306e\u9ad8\u89e3\u50cf\u5ea6\u7248\u3001\u7121\u4fee\u6b63\u7248\uff08\u4e00\u90e8\u306e\u307f\uff09\u304c\u30e1\u30a4\u30f3\u3068\u306a\u308a\u307e\u3059\u3002\r\n\u4e00\u90e8\u8131\u8863\u5dee\u5206\u3084\u5225\u30a2\u30f3\u30b0\u30eb\u3001FANBOX\u9650\u5b9a\u306e\u30b7\u30fc\u30f3\u3082\u5c11\u3057\u8ffd\u52a0\u3067\u6295\u7a3f\u3057\u3066\u3044\u307e\u3059\u3002\r\nThe images in the plan are not much different from the my published on pixiv,\r\nmainly the high resolution version, uncensored version.\r\n(Penetration scenes and male genitalia are censored in accordance with Japanese law.)\r\n\r\n\u3054\u652f\u63f4\u3092\u9802\u3044\u305f\u5834\u5408\u306fPC\u30cf\u30fc\u30c9\u3084\u30bd\u30d5\u30c8\u30a6\u30a7\u30a2\u306a\u3069\u5236\u4f5c\u74b0\u5883\u3092\u6574\u3048\u308b\u305f\u3081\u306b\u4f7f\u7528\u3055\u305b\u3066\u3044\u305f\u3060\u304d\u307e\u3059\u3002\r\nIf I get support, I use it to purchase a production environment such as PC hardware or software.\r\n\r\n\u3042\u3068marugaogaski\u304c\u5c11\u3057\u5143\u6c17\u306b\u306a\u308a\u307e\u3059\u3002\r\nAnd my motivation goes up.\r\n\r\n\u9ad8\u89e3\u50cf\u5ea6\u7248\u30b5\u30f3\u30d7\u30eb\r\nHighResolution Sample\r\nhttps:\/\/marugaogaski.fanbox.cc\/posts\/577459\r\nhttps:\/\/marugaogaski.fanbox.cc\/posts\/1124127",
            "imageUrl":"https:\/\/pixiv.pximg.net\/c\/520x280_90_a2_g5\/fanbox\/public\/images\/creator\/28061721\/cover\/09CDfIBsX8xm4zy22oz4gtSa.jpeg",
            "imageUrlMobile":"https:\/\/pixiv.pximg.net\/c\/520x280_90_a2_g5\/fanbox\/public\/images\/creator\/28061721\/cover\/09CDfIBsX8xm4zy22oz4gtSa.jpeg",
            "hasAdultContent":true
         }
      ],
      "bookmarkCount":{
         "public":{
            "illust":1,
            "novel":0
         },
         "private":{
            "illust":0,
            "novel":0
         }
      },
      "externalSiteWorksStatus":{
         "booth":false,
         "sketch":false,
         "vroidHub":false
      },
      "request":{
         "showRequestTab":false,
         "postWorks":{
            "artworks":[
               
            ],
            "novels":[
               
            ]
         }
      }
   }
}
```

**Comment by sakkamade** [comment]
```
A JavaScript error occurred in the main proccess
```
I'm experience this problem either. 

It happens occasionally, when I'd click on `"Pixiv Omina"` button of `Pixiv Toolkit` in browser window (Firefox, Windows 10). 
Most of the times restart of `Pixiv Omina` helps to solve it.

The problem never taking a place when I'd paste the link directly into `Pixiv Omina`.

Cannot provide a work, cause that occurred quite a while ago.

**Comment by leoding86** [comment]
@roshiajin-kunai Cannot reproduce the issue either. Need more informations.

**Comment by sakkamade** [comment]
I believe this error's occuring after a lot of works has been downloaded, it has nothing to do with a specific artwork. (Because sometimes the error is fixeing just by clicking the button of the `P Toolkit` one more time.)
It's not an application's error, it's just a website's restrictions.

(Just my assumption.)

**Update:**
I was able to reproduce the issue (once): 

- minimize `Pixiv Omina`, wait some time (>30 minutes) 
- you can start some applications, 'till ~95% RAM (so, the `Pixiv Omina`will be unloaded from RAM)
- ~~click `Omina` button in the browser.~~ just parsing link into `Pixiv Omina` is enough

https://www.pixiv.net/en/artworks/48229930

**Comment by Ventuquies** [comment]
same for me, it seems that if too much was processed there is either an error with the login process or a restriction with the download. It works after some time again, but sometimes it is just try&error until it works again

**Comment by sakkamade** [comment]
**Update-2:**

After restarting `Pixiv Omina` it shows the message `You need to login first` and the second after that, it's begin to download files which were added in previous session.

So the issue that's `Pixiv Omina` loosing connection with Pixiv, during unloading from RAM and doesn't reconnect afterwards.

**Comment by leoding86** [comment]
JSON parse issue was found and fixed in version 0.5.1. Sorry for late fixing. The issue caused by a typo which cause a module loaded wrong settings.
